client.go

```diff
@@ -0,0 +1,277 @@
+package globalsign
+
+import (
+	"bytes"
+	"context"
+	"crypto/tls"
+	"encoding/json"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"net/http"
+	"net/url"
+	"sync"
+	"time"
+)
+
+// Client GlobalSign sdk client.
```

Comment:
This is a broken sentence, should improve it... Is this also used externally?
We can unexport anything not needed externally